client/jetpack-cloud/components/layout/top.tsx

```diff
@@ -5,9 +5,5 @@ type Props = {
 };
 
 export default function LayoutTop( { children }: Props ) {
-	return (
-		<div className="jetpack-cloud-layout__top">
-			<div className="jetpack-cloud-layout__top-wrapper">{ children }</div>
-		</div>
-	);
```

Comment:
This done to solve the flickering issue when switching between the sticky & non-sticky content